server/package.json

```diff
@@ -11,8 +11,6 @@
     "dotenv": "^4.0.0",
     "express": "^4.16.2",
     "joi": "^17.2.1",
-    "lodash": "^4.17.20",
```

Comment:
These dependencies were moved to the main folder to reduce unnecessary repetition as they are used in both the react-ui and server codebases.